### PR TITLE
ci(release): Move to getsentry/publish for releases

### PR DESCRIPTION
**Pull request by BYK**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -32,36 +24,26 @@ jobs:
...
-          fetch-depth: 20
+          token: ${{ secrets.GH_RELEASE_PAT }}
       - uses: getsentry/craft@master
         name: Craft Prepare
-        if: ${{ !github.event.inputs.skip_prepare }}
         with:
           action: prepare
           version: ${{ env.RELEASE_VERSION }}
         env:
           ZEUS_API_TOKEN: ${{ secrets.ZEUS_API_TOKEN }}
```
Comment (chadwhitacre) [inline .github/workflows/release.yml:34]: Why do we still need the Zeus token here? Isn't Zeus used for publishing, and we're no longer publishing from here?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -32,36 +24,26 @@ jobs:
...
-          ./scripts/bump-version.sh '' $(date -d "$(echo $RELEASE_VERSION | sed -e 's/^\([0-9]\{2\}\)\.\([0-9]\{1,2\}\)\.[0-9]\+$/20\1-\2-1/') 1 month" +%y.%-m.0.dev0)
-          git diff --quiet || git commit -anm 'meta: Bump new development version' && git pull --rebase && git push
+          # TODO: Replace token with GH_RELEASE_PAT after verification
+          token: ${{ secrets.GH_SENTRY_BOT_PAT }}
+          script: |
+            const repoInfo = context.repo;
+            await github.issues.createComment({
+              owner: repoInfo.owner,
+              repo: 'publish',
+              title: `publish ${repoInfo.repo}@{process.env.RELEASE_VERSION}`,
```
Comment (chadwhitacre) [inline .github/workflows/release.yml:46]: Ship has kinda sailed, but it'd be easier parsing if we just used whitespace vs `@` here:

```
deploy sentry 20.12.0
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -32,36 +24,26 @@ jobs:
...
+              // TODO: Remove the following line after verification
+              labels: ['dry-run'],
```
Comment (chadwhitacre) [inline .github/workflows/release.yml:48]: 👀

File: scripts/post-release.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu
+
+git checkout master
+./scripts/bump-version.sh '' $(date -d "$(echo $RELEASE_VERSION | sed -e 's/^\([0-9]\{2\}\)\.\([0-9]\{1,2\}\)\.[0-9]\+$/20\1-\2-1/') 1 month" +%y.%-m.0.dev0)
+git diff --quiet || git commit -anm 'meta: Bump new development version' && git pull --rebase && git push
```
Comment (chadwhitacre) [inline scripts/post-release.sh:6]: Will we DRY this up somehow for the next repo we integrate?

**Review by chadwhitacre — APPROVED** [review]
Some q's, no blockers.

Did not test.

File: .github/workflows/release.yml (hunk 1 above)
Comment (BYK) [reply .github/workflows/release.yml:34]: ~~Zeus is also to upload artifacts but let me check.~~

Seems like you are right, I'll axe it too.

File: .github/workflows/release.yml (hunk 2 above)
Comment (BYK) [reply .github/workflows/release.yml:46]: This is the notation used by `npm` and in some other places too, that's why I preferred it.

File: .github/workflows/release.yml (hunk 3 above)
Comment (BYK) [reply .github/workflows/release.yml:48]: The thing is, to be able to add labels to issues, you need push permissions to the repo. Hence the comment. After all, maybe I should have gone with your "no-dry-run" label 😛

File: scripts/post-release.sh (hunk 4 above)
Comment (BYK) [reply scripts/post-release.sh:6]: This one only applies to Sentry so no need for that. This is why we introduced the support for `post-release.sh` script in Craft and not made something similar the default.

File: .github/workflows/release.yml (hunk 3 above)
Comment (chadwhitacre) [reply .github/workflows/release.yml:48]: Yup, I wuz just noticing what we talked about earlier. ☺️

**Pull request merged**